packages/cli-repl/test/e2e.spec.ts

```diff
@@ -280,11 +280,20 @@ describe('e2e', function() {
 
     let shell;
     beforeEach(async() => {
-      shell = TestShell.start({ args: [ '--nodb' ] });
+      // We set CLEAR_SIGINT_LISTENERS to remove all `process.on('SIGINT')`
```

Comment:
I'd do this in TestShell itself only if the binary was not passed: we run e2e tests on the built binary too which will make the coverage meaningless and removing listeners will possibly taint the tests.  Would be nice to leave everything intact in that case.

Reply:
> and removing listeners will possibly taint the tests

Yeah, I've thought about this tainting the test but it just seems highly unlikely that this might ever become an issue. I still like the suggestion of doing this only if we're not running the executable, that seems just fine to me :+1:

Reply:
done :+1: